moto/emr/responses.py

```diff
@@ -1418,3 +1457,36 @@ def remove_auto_scaling_policy(self) -> str:
     <RequestId>2690d7eb-ed86-11dd-9877-6fad448a8419</RequestId>
   </ResponseMetadata>
 </DeleteSecurityConfigurationResponse>"""
+
+PUT_BLOCK_PUBLIC_ACCESS_CONFIGURATION_TEMPLATE = """<PutBlockPublicAccessConfigurationResponse xmlns="http://elasticmapreduce.amazonaws.com/doc/2009-03-31">
+  <ResponseMetadata>
+    <RequestId>2690d7eb-ed86-11dd-9877-6fad448a8419</RequestId>
+  </ResponseMetadata>
+</PutBlockPublicAccessConfigurationResponse>"""
+
+GET_BLOCK_PUBLIC_ACCESS_CONFIGURATION_TEMPLATE = """
+  <GetBlockPublicAccessConfigurationResponse xmlns="http://elasticmapreduce.amazonaws.com/doc/2009-03-31">
+    <GetBlockPublicAccessConfigurationResult>
+      <BlockPublicAccessConfiguration>
+        <BlockPublicSecurityGroupRules>
+          {{block_public_security_group_rules}}
+        </BlockPublicSecurityGroupRules>
+        <BlockPublicSecurityGroupRuleRanges>
+          {% for rule_range in permitted_public_security_group_rule_ranges %}
+            <BlockPublicSecurityGroupRuleRange>
+              <MinRange>{{rule_range['min_range']}}</MinRange>
+              <MaxRange>{{rule_range['max_range']}}</MaxRange>      
+            </BlockPublicSecurityGroupRuleRange>
+          {% endfor %}
+        </BlockPublicSecurityGroupRuleRanges>
+      </BlockPublicAccessConfiguration>
+      <BlockPublicAccessConfigurationMetadata>
+        <CreationDateTime>{{creation_date_time}}</CreationDateTime>
+        <CreatedByArn>{{created_by_arn}}</CreationByArn>
```

Comment:
This XML is mismatched - that's the next error after fixing the `Unknown type structure` error :slightly_smiling_face: